cmd/cli/kubectl-kyverno/processor/result.go

```diff
@@ -46,15 +48,26 @@ func (rc *ResultCounts) addEngineResponse(auditWarn bool, response engineapi.Eng
 						case engineapi.RuleStatusPass:
 							rc.pass++
 						case engineapi.RuleStatusFail:
+							auditWarning := false
 							if !scored {
 								rc.warn++
 								break
 							} else if auditWarn && response.GetValidationFailureAction().Audit() {
+								auditWarning = true
 								rc.warn++
 							} else {
 								rc.fail++
 							}
+							if !policyReport {
+								if auditWarning {
+									fmt.Printf("\npolicy %s -> resource %s failed as audit warning: \n", policy.GetName(), resourcePath)
+								} else {
+									fmt.Printf("\npolicy %s -> resource %s failed: \n", policy.GetName(), resourcePath)
+								}
+								fmt.Printf("%d. %s: %s \n", i+1, valResponseRule.Name(), valResponseRule.Message())
+							}
 						case engineapi.RuleStatusError:
+							fmt.Printf("\npolicy %s -> resource %s error: %s\n", policy.GetName(), resourcePath, valResponseRule.Message())
```

Comment:
`validate.message` is only printed on failures as it implies that the resource violates the rule. I checked the `release-1.10` branch, and there is no message displayed on errors. It can be removed.

Reply:
Done